scripts/start.ts

```diff
@@ -20,6 +20,26 @@ const GRAPHQL_EXTENSION = ".graphql.json";
 const allOpenAPIPaths: string[] = [];
 const allGraphqlPaths: string[] = [];
 
+function processObj(obj: any) {
```

Comment:
`processObj` is a very generic name. What do you think about naming it with details about the problem which it's solving?

ChatGPT might help with that

![image](https://github.com/deco-cx/apps/assets/18706156/2de04ee3-77f6-432d-9e9f-0de9149915b0)

Reply:
Beware that this function appears to be modifying the input!